src/components/Popper/PopperContent.tsx

```diff
@@ -155,9 +155,7 @@ const PopperContent = (
                     />
                 ))}
             <Surface elevation={2} style={containerStyle} {...rest} ref={ref} onLayout={onUpdate}>
-                <Text style={contentTextStyles} numberOfLines={1}>
-                    {restElements}
-                </Text>
+                <Text style={contentTextStyles}>{restElements}</Text>
```

Comment:
Why shouldn't we completely remove the Text wrapper?

If we don't want to remove it, we should expose its props

Reply:
Yeah, showed Shashank, seems there's a dependency for that with some other components like Tooltip.

Reply:
For now it works, but we should investigate this further though, not sure wrapping views in text is a good idea.

Reply:
Okay but we should at least expose the props.
With this, we can't modify the wrapper

Reply:
you have this so that you can add a default color to the data.
Maybe we should simply wrap this in a ThemeProvider with Text theme overwritten.. That way, it still work

Reply:
Yeah, I'll expose the numberOfLines prop